src/Utilities/ToolTask.cs

```diff
@@ -1715,6 +1715,41 @@ public override bool Execute()
         } // Execute()
 
         /// <summary>
+        /// Replace backslashes with OS-specific path separators,
+        /// except when likely that the backslash is intentional.
+        /// </summary>
+        /// <remarks>
+        /// Not a static method so that an implementation can
+        /// override with more-specific knowledge of what backslashes
+        /// are likely to be correct.
+        /// </remarks>
+        virtual protected string ReplaceBackslashes(string input)
```

Comment:
Now that it's a public type, I wonder if  `NormalizeString` might be a better name. Or `NormalizeToolArguments`